buildSrc/build.gradle

```diff
@@ -1,7 +1,6 @@
-
 plugins {
     id 'java'
```

Comment:
old Lombok processor no longer maintained, and doesn't work with JDK 17.  Upgrade to IntelliJ recommended one.  Works fine w/o code changes (other than build.gradle code)